usaspending_api/common/helpers.py

```diff
@@ -28,6 +28,27 @@ def get_params_from_req_or_request(request=None, req=None):
     return params
 
 
+def generate_fiscal_year(context):
```

Comment:
Can we alter the parameter names for this function and the 2 below? if it's a date object coming in, it should be named appropriately.

Reply:
can do